pkg/setting/setting.go

```diff
@@ -1571,8 +1571,9 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 
 	cfg.OAuthCookieMaxAge = auth.Key("oauth_state_cookie_max_age").MustInt(600)
 	cfg.SignoutRedirectUrl = valueAsString(auth, "signout_redirect_url", "")
+
 	// Deprecated
-	cfg.OAuthSkipOrgRoleUpdateSync = auth.Key("oauth_skip_org_role_update_sync").MustBool(false)
+	cfg.OAuthSkipOrgRoleUpdateSync = true
```

Comment:
always on

Reply:
Does it make sense to explain what is replacing it and why it is hardcoded to true?

Reply:
Also can we perhaps test somewhere that this flag is always set to true, as that is the expectation at the moment? Maybe in the OAuth tests?

Reply:
we could remove at as part of the configuration. I wanted to have it there to be able to warn. But it might be better to just remove it entirely